controllers/idler/app_idler.go

```diff
@@ -0,0 +1,218 @@
+package idler
```

Comment:
IMHO, the file should be called `aap_idler.go`. Typo?

Reply:
Yes! :)

Reply:
Renamed in https://github.com/codeready-toolchain/member-operator/pull/631/commits/20fa50f5aaf38e97a80ee572071d3c219bd10914